### ISSUE TITLE
[eslint-plugin-lit-a11y]: role-has-required-aria-attrs in conflict with official W3org suggestion

### ISSUE THREAD

**Issue by a-kosanovic** [open-wc/open-wc#2506]
## Expected behavior
- https://www.w3.org/WAI/ARIA/apg/patterns/switch/
> If the switch element is an HTML input[type="checkbox"], it uses the HTML checked attribute instead of the aria-checked property.

- open-wc should be using the latest version (v6.6.1)  of 'eslint-plugin-lit-a11y' package where this standard is supported 

## Actual Behavior
- Getting error: `The "switch" role requires the attribute "aria-checked".` when following `w3.org` standards 
- more details here: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/900
- `[eslint-plugin-lit-a11y]` plugin is not up to date with latest version 

## Additional context
- using `"@open-wc/testing": "3.1.6",`

Thank you :pray:
